src/app.ts

```diff
@@ -18,8 +18,8 @@ async function main() {
     const logger = new PluginLogger();
     const config: PluginConfig = defaultPluginConfig;
     populatePluginConfigFromEnv(config, env);
-
     if (!config.rendering.chromeBin && (process as any).pkg) {
+      //@ts-ignore
```

Comment:
This seems like seem issue with types, `executablePath` works but TS complains that it doesn't exist.